queue_model_objects_v1.py

```diff
@@ -241,11 +241,9 @@ def get_display_name(self):
             return ''
 
     def init_from_sc_sample(self, sc_sample):
-        #self.loc_str = ":".join(map(str,sc_sample[-1]))
-        #self.location = sc_sample[-1]
-        #self.set_name(self.loc_str)
-        self.loc_str = str(sc_sample[1]) + ':' + str(sc_sample[2])
-        self.location = (sc_sample[1], sc_sample[2])
+        self.loc_str = ":".join(map(str,sc_sample[-1]))
```

Comment:
Hi,
No, that code does not work with the master branch. The problem is in init_from_sc_sample and the argument sc_sample. Could we once define that list? Otherwise we will return to this function again and again
Now in 2.2 the list looks like this: ('barcode, None, None, 'Sample name', (5, 5)). I am not sure why those two None are necessary. Could we have something like this: (barcode, name, (puck_id, sample_id))?